### ISSUE TITLE
Logic Apps (Preview) templates fail to load in VSCode

### ISSUE THREAD

**Issue by aroonk1982** [Azure/logicapps#210]
Does this occur consistently?  Yes
Repro steps:

1. Using the new Logic Apps (Preview) extension in Visual Studio Code, create a new Project
2. Select folder for new project
3. Below error occurs when VS Code is looking for templates

Action: azureLogicAppsV2.createNewProject
Error type: Error
Error Message: Failed to get latest templates: Unexpected token E in JSON at position 0

Version: 0.0.5
OS: win32
OS Release: 10.0.18363
Product: Visual Studio Code
Product Version: 1.51.0
Language: en

<details>
<summary>Call Stack</summary>

```
CentralTemplateProvider.<anonymous> extension.bundle.js:164:430605
fulfilled extension.bundle.js:164:427159
processTicksAndRejections task_queues.js:94:5
```

</details>

**Comment by tonytang-microsoft-com** [comment]
Thanks @aroonk1982 We are preparing a fix for this issue and we are planning to release a new version ASAP.

**Comment by vikash-anand** [comment]
@aroonk1982  We have released a new version of the extension - v0.0.6 today which should fix this issue. Please update your extension and let us know if the issue exists.

**Comment by nidubey1** [comment]
@aroonk1982 I have tested the same and it works fine now. Reference Document: https://docs.microsoft.com/en-us/azure/logic-apps/create-single-tenant-workflows-visual-studio-code
